### ISSUE TITLE
Apple clang now requires minimal deployment target for 32-bit targets

### ISSUE THREAD

**Issue by mobileben** [KjellKod/g3log#271]
I recently updated my fork with this branch. Building now results in an error

> clang: error: invalid iOS deployment version '-miphoneos-version-min=11.3', iOS 10 is the maximum deployment target for 32-bit targets [-Winvalid-ios-deployment-target]

I do have a fix for it. I'll be posting a pull request for this shortly.

**Comment by KjellKod** [comment]
Looking forward to it

**Comment by mobileben** [comment]
BTW, I'm working on the initial release for the project I've been working on for the past year. I know I owe you looking at unit testing for iOS.

Oh, also, I noticed that when I run in release mode, I get some weird characters in the output. Have you ever seen anything like this? I figure it has to be in my sink. Just curious ...

Example (the \353\377)

2018/06/05 20:53:15 426249	\353\377 [JKSContentManager.m->+[JKSContentManager downloadContent:dstUrl:manifestUrl:unpackUrl:md5:progress:completion:]_block_invoke:689]	Verifying md5

**Comment by KjellKod** [comment]
Congrats on the project :)

If you use a custom sink make sure that logging level string value is outputted. It almost look like the object and not the string is being pushed to the logging line. I don’t know why the compiler isn’t complaining about that though...

**Comment by KjellKod** [comment]
Addressed in https://github.com/KjellKod/g3log/pull/272#pullrequestreview-126370720

**Comment by mobileben** [comment]
@KjellKod thanks. Not quite done but close!

Figured out my issue. It turns out I used another function to generate the LEVELS. I did this so I could tack on my own additions. However, since LogCapture takes the arg by const ref. This is no bueno since I passed back by value.

LEVELS toG3Level(Logger::Level level);

This was the function I was using. I changed it to return a const ref and it worked. Its fine to do this as it returns g3log's INFO, DBG, WARNING, FATAL as well as my own private LEVELS which are also const. 

Thanks for the speculation on the problem ... pointed me in the right direction!
